docs/bugs.json

```diff
@@ -1,4 +1,15 @@
 [
+    {
+        "name": "ByteOptimizationBug",
+        "summary": "Byte expressions that contain the constant 31 as the second argument can result in unexpected values.",
```

Comment:
The optimizer incorrectly handles `byte` opcodes that use the constant 31 as second argument. This can happen when performing index access on `bytesNN` types with a compile-time constant value (not index) of 31.